std/experimental/ndslice/slice.d

```diff
@@ -1767,6 +1767,185 @@ struct Slice(size_t _N, _Range)
         assert(a != [[9, 2], [3, 4]]);
     }
 
+    /++
+    Computes hash value using MurmurHash3 algorithms without finalization step.
```

Comment:
> without finalization step

"without the finalization step"

You should add "the" before the other mentions of finalization step.